portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -45,6 +47,16 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     infoAboutMe.clear();
   }
 
+  // This function handles user comments whenever the user hits the "submit" button on the webpage.
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the input from the form.
+    String text = request.getParameter("text-input");
+    userComments.add(text);
+    // Send the user back to the page after submitting a comment
+    response.sendRedirect("/index.html");
```

Comment:
It looks like this refreshes the page, and when that happens, I'm not seeing my comment show up :(

Reply:
I have implemented it. Let me know how it works :)

Reply:
It works now, thanks!

Reply:
From a UX perspective, I think it would be great if we could update this without it refreshing the page. But up to you if that's a change you are interested in implementing!